addons/sale_stock/models/sale_order.py

```diff
@@ -207,6 +207,9 @@ def action_view_delivery(self):
         return action
 
     def action_cancel(self):
+        cancel_warning = self._show_cancel_wizard()
+        if cancel_warning:
+            return super(SaleOrder, self).action_cancel()
```

Comment:
You could just call the super before.

Reply:
@amoyaux thanks for the review!
Could you check the update ?